docs/usage_guide.rst

```diff
@@ -716,13 +716,13 @@ Rotki Generic Trades Import
 This is solely for importing generic trades. The expected file format is **CSV** with the following headers and description as a guide:
 
 1. **Location**: This is the source of the data. It should be one of Rotki's `supported locations`_. If it is not supported, use ``"external"``.
-2. **Base Currency**: The currency bought in a ``"Buy"`` and sold in a ``"Sell"``. e.g BTC/ETH where BTC is the base currency.
-3. **Quote Currency**: The currency sold in a ``"Buy"`` and bought in a ``"Sell"`` e.g BTC/ETH where ETH is the quote currency.
+2. **Base Currency**: The currency bought in a ``"Buy"`` and sold in a ``"Sell"``. e.g BTC/ETH where BTC is the base currency. Is important to use the identifier of the asset used in rotki or otherwise the row won't be read.
```

Comment:
```suggestion
2. **Base Currency**: The currency bought in a ``"Buy"`` and sold in a ``"Sell"``. e.g BTC/ETH where BTC is the base currency. It is important to use the identifier of the asset used in rotki or otherwise the row won't be read.
```

same for other places it is used.

Reply:
done, thanks